changelog/8913.feature.md

```diff
@@ -0,0 +1,46 @@
+Introduces a new policy called [`UnexpecTEDIntentPolicy`](./policies.mdx#unexpected-intent-policy).
+
+`UnexpecTEDIntentPolicy` helps you [review conversations](./conversation-driven-development.mdx#review)
+ and also allows your bot to react to unexpected user turns in conversations.
+ It is an auxiliary policy that should only be used in conjunction with
+ at least one other policy, as the only action that it can trigger
+ is the special and newly introduced
+ [`action_unlikely_intent`](./default-actions.mdx#action_unlikely_intent) action.
+
+ The auto-configuration will include `UnexpecTEDIntentPolicy` in your
+ configuration automatically, but you can also include it yourself
+ in the `policies` section of the configuration:
+
+ ```
+ policies:
+   - name: UnexpecTEDIntentPolicy
+     epochs: 200
+     max_history: 5
+ ```
+
+As part of the feature, it also introduces:
+
+- [`IntentMaxHistoryTrackerFeaturizer`](./policies.mdx#3-intent-max-history)
+  to featurize the trackers for `UnexpecTEDIntentPolicy`.
+- `MultiLabelDotProductLoss` to support `UnexpecTEDIntentPolicy`'s multi-label training objective.
+- A new default action called [`action_unlikely_intent`](./default-actions.mdx#action_unlikely_intent).
+
+
+`rasa test` command has also been adapted to `UnexpecTEDIntentPolicy`:
+
+- If a test story contains `action_unlikely_intent` and the policy ensemble does not trigger it, this leads to
+  a test error (wrongly predicted action) and the corresponding story will be logged in `failed_test_stories.yml`.
+- If the story does not contain `action_unlikely_intent` and Rasa Open Source does predict it then
+  the prediction of `action_unlikely_intent` will be ignored for the evaluation (and hence not lead
+  to a prediction error) but the story will be logged in a file called `stories_with_warnings.yml`.
+
+
+`rasa data validate` command should warn if `action_unlikely_intent` is
+included in the training stories. Accordingly, `YAMLStoryWriter` and `MarkdownStoryWriter` have been updated to not dump `action_unlikely_intent` when writing stories to a file.
+
+:::caution
+The introduction of a new default action makes the release **breaking backwards compatibility of machine learning models**.
```

Comment:
The introduction of a new default action **brakes backward compatibility of previously trained models**